apps/dashboard/test/system/batch_connect_test.rb

```diff
@@ -42,10 +42,10 @@ def setup
   end
 
   test 'changing node type changes mins & maxs' do
-    # max starts out at 20
+    # max starts out at 7
     visit new_batch_connect_session_context_url('sys/bc_jupyter')
-    assert_equal 20, find_max('bc_num_slots')
-    assert_equal 1, find_min('bc_num_slots')
+    assert_equal 7, find_max('bc_num_slots')
+    assert_equal 3, find_min('bc_num_slots')
```

Comment:
Note that all these tests change because, well the defaults 20 & 1, are being overwritten by `any`'s settings 7 & 3.